definitions/linux.yaml

```diff
@@ -533,4 +533,15 @@ sources:
 - type: FILE
   attributes: {paths: ['/var/log/kern.log*']}
 labels: [Logs]
-supported_os: [Linux]
\ No newline at end of file
+supported_os: [Linux]
+---
+name: DSDTTable
```

Comment:
seeing this is linux specific artifact please prefix the name with Linux as: LinuxDSDTTable